play/mbta/mzcompose.yml

```diff
@@ -7,80 +7,33 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-x-port-mappings:
-  - &kafka ${KAFKA_PORT:-9092:9092}
-  - &materialized ${MZ_PORT:-6875:6875}
-
 version: '3.7'
 services:
+  mbta-demo:
+    mzbuild: mbta-demo
+    volumes: [.:/workdir]
+    depends_on: [kafka]
+
   materialized:
     mzbuild: materialized
     command: --disable-telemetry
     environment:
       - MZ_DEV=1
-    ports:
-      - *materialized
-    volumes:
-      - .:/workdir
+    ports: [6875]
+    volumes: [.:/workdir]
+
   zookeeper:
```

Comment:
Are we keeping the definitions for "zookeeper" and "kafka" in the yml file , even though we have them in `mzcompose.py` ?

Reply:
Yeah, my current thinking is that the stuff in `demo/` and `play/` that is meant to be user-facing should continue to define services in mzcompose.yml so they're a bit more familiar to users. Gonna sync with the devex team in the new year about this though; possibly mzcompose is too much machinery period and we should abandon hope of making it understandable to non-employees.